src/test/regress/expected/create_drop_database_propagation.out

```diff
@@ -932,6 +928,259 @@ SELECT * FROM public.check_database_on_all_nodes('test_db') ORDER BY node_type;
+ worker node (remote) | {"database_properties": {"datacl": ["=Tc/non_propagated_role", "non_propagated_role=CTc/non_propagated_role", "propagated_role=c/non_propagated_role"], "datname": "test_db", "datctype": "C", "encoding": "UTF8", "datcollate": "C", "tablespace": "pg_default", "daticurules": null, "datallowconn": true, "datconnlimit": -1, "daticulocale": null, "datistemplate": false, "database_owner": "non_propagated_role", "datcollversion": null, "datlocprovider": "c"}, "pg_dist_object_record_for_db_exists": true, "stale_pg_dist_object_record_for_a_db_exists": false}
+ worker node (remote) | {"database_properties": {"datacl": ["=Tc/non_propagated_role", "non_propagated_role=CTc/non_propagated_role", "propagated_role=c/non_propagated_role"], "datname": "test_db", "datctype": "C", "encoding": "UTF8", "datcollate": "C", "tablespace": "pg_default", "daticurules": null, "datallowconn": true, "datconnlimit": -1, "daticulocale": null, "datistemplate": false, "database_owner": "non_propagated_role", "datcollversion": null, "datlocprovider": "c"}, "pg_dist_object_record_for_db_exists": true, "stale_pg_dist_object_record_for_a_db_exists": false}
+(3 rows)
+
+REVOKE CONNECT ON DATABASE test_db FROM propagated_role;
+DROP DATABASE test_db;
+DROP ROLE propagated_role, non_propagated_role;
+-- test citus_internal_acquire_citus_advisory_object_class_lock with null input
+SELECT citus_internal_acquire_citus_advisory_object_class_lock(null);
+ERROR:  object_class cannot be NULL
+SELECT citus_internal_acquire_citus_advisory_object_class_lock(26); -- OCLASS_DATABASE
+ citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus_internal_acquire_citus_advisory_object_class_lock(10); -- another valid OCLASS
+ citus_internal_acquire_citus_advisory_object_class_lock
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus_internal_acquire_citus_advisory_object_class_lock(-1); -- invalid OCLASS
+ERROR:  invalid object class: -1
+SELECT citus_internal_acquire_citus_advisory_object_class_lock(100); -- invalid OCLASS
+ERROR:  invalid object class: 100
+SELECT 1 FROM run_command_on_all_nodes('ALTER SYSTEM SET citus.enable_create_database_propagation TO ON');
+ ?column?
+---------------------------------------------------------------------
+        1
+        1
+        1
+(3 rows)
+
+SELECT 1 FROM run_command_on_all_nodes('SELECT pg_reload_conf()');
+ ?column?
+---------------------------------------------------------------------
+        1
+        1
+        1
+(3 rows)
+
+SELECT pg_sleep(0.1);
+ pg_sleep
+---------------------------------------------------------------------
+
+(1 row)
+
+-- only one of them succeeds and we don't run into a distributed deadlock
+SELECT COUNT(*)=1 FROM run_command_on_all_nodes('CREATE DATABASE concurrent_create_db') WHERE success;
```

Comment:
For debugging purposes it's nicer to show the actual value instead of true/false
```suggestion
SELECT COUNT(*) FROM run_command_on_all_nodes('CREATE DATABASE concurrent_create_db') WHERE success;
```